VndbSharp/Structs/Models/VnList/VnList.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace VndbSharp.Structs.Models.VnList
+{
+    [JsonObject]
+    public class VnList
+    {
+        [JsonProperty("vn")]
+        public Int32 VisualNovelId;
+        [JsonProperty("status")]// : TODO: Possibly change this to a byte, since only 4 values will be returned?
+        public Int32 Status;
+        [JsonProperty("added")]
+        public Int32 TimeAdded;
```

Comment:
The UnixTimestamp Converter can also be applied [here](https://github.com/micah686/VndbSharp/blob/1f346062febd17b5eaac3a6ad13388b38108d2d6/VndbSharp/Structs/Models/Votelist/Votelist.cs#L18), as the comment mentions.